content/docs/howto/java.md

```diff
@@ -528,6 +528,25 @@ jcmd 1 VM.native_memory summary
 
 The first argument should be the JVM PID, in the case of the Paketo Java buildpack, this will be `1`.
 
+### Java NMT and Java 8
+
+At the moment, there is [a bug in Java 8](https://github.com/paketo-buildpacks/bellsoft-liberica/issues/131) that prevents Java NMT from being used. If you attempt to use it with Java 8 in a buildpack generated container, the JVM will crash on startup. The Java buildpack has disabled Java NMT when you are using Java 8, so users are not impacted by the bug. At the same time, it is not presently possible to enable Java NMT if you are running Java 8.
+
+A fix for this issue has been submitted upstream to OpenJDK. It will take time for this fix to be accepted and picked up by all of the Java vendors. We are monitoring this situation, and as Java vendors pick up this fix we will re-enable Java NMT support for Java 8 users in the buildpack.
+
+### Java NMT and Reloadable Processes
+
+Java NMT support does not work with [Reloadable Processes](/#enable-process-reloading). This is because when using reloadable processes, the JVM PID is not 1 and not easily determinable by the buildpack Java NMT helper. Thus the helper cannot set the environment variables required by the JVM to enable Java NMT.
```

Comment:
```suggestion
Java NMT support does not work with [Reloadable Processes]({{< relref "#enable-process-reloading" >}}). This is because when using reloadable processes, the JVM PID is not 1 and not easily determinable by the buildpack Java NMT helper. Thus the helper cannot set the environment variables required by the JVM to enable Java NMT.
```